### ISSUE TITLE
Setting templates

### ISSUE THREAD

**Issue by lionel-** [joostkremers/pandoc-mode#58]
Currently the template option is set up to expand a full path. This makes it difficult to use a template from the pandoc templates folder. E.g. `(pandoc--set 'template "mycustomtemplate")` will result in a failure.

Is there a workaround for this? If not, would you consider removing the `'full-path` qualifier for that option?

**Comment by joostkremers** [comment]
Yes, I see the problem. Pandoc searches in its template directory if `--template` does not specify a fully qualified path, so I guess `pandoc-mode` should allow this as well. I'll need to think about how to best implement this, though. Just removing the `full-path` qualifier doesn't seem like the best idea, since this problem probably exists with other file options as well.

**Comment by joostkremers** [comment]
I did some work reorganising the way file options are handled. Now by default, every file option is stored as a relative path, so if you set the template file and type `mycustomtemplate` or something, that's exactly what's stored. If you want to store an absolute path, you can use the prefix argument `C-u`.

Also, if you use `pandoc--set`, as you do, relative file names are not expanded anymore. Only file names that are absolute are expanded. That should solve your problem.

Let me know if you run into problems.
